lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -1112,9 +1232,9 @@ bool mlir::triton::preProcessLoopAndGetSchedule(
   // a scaffold for the final schedule.
   DenseSet<Operation *> rootUsers;
   tt::CoarseSchedule coarseSchedule(numStages);
-  llvm::MapVector<Operation *, LoadInfo> loadToInfo =
-      scheduleLoads(forOp, coarseSchedule, rootUsers, numStages);
-  if (loadToInfo.empty())
+
+  scheduleLoads(forOp, coarseSchedule, rootUsers, numStages);
+  if (coarseSchedule.opToStageAndCluster.size() == 0)
```

Comment:
this is (badly encapsulated) piece of implementation detail that I tried not to leak so far. Can we have `empty()` helper perhaps?

Reply:
You mean empty helper for CoarseSchedule? Sure!

Reply:
Done adding the helper function.

Reply:
Thank you!